### ISSUE TITLE
getCryptoData is an undefined name

### ISSUE THREAD

**Issue by cclauss** [alexanderepstein/cryptowatch#2]
**Issue Label:**
* [x] Bug
* [ ] New feature
* [ ] Enhancement

**Description:**
Undefined names can raise NameError at runtime.

flake8 testing of https://github.com/alexanderepstein/cryptowatch on Python 3.6.2

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./cryptoUtils/cryptoUtils.py:198:28: F821 undefined name 'getCryptoData'
            exchangeRate = getCryptoData(response, "ER")
                           ^

./cryptoUtils/cryptoUtils.py:204:28: F821 undefined name 'getCryptoData'
            exchangeRate = getCryptoData(response, "ER")
                           ^

./cryptoUtils/cryptoUtils.py:210:28: F821 undefined name 'getCryptoData'
            exchangeRate = getCryptoData(response, "ER")
                           ^
```

**Comment by alexanderepstein** [comment]
I'm curious why my computer hasn't thrown an error about this but maybe its because I have a version of cryptowatch installed with getCryptoData still defined. Thanks for the help :+1: 

 Fixed in this commit 0160d8a354417adc42c8b46db82a2eb7cb15cb97

**Comment by alexanderepstein** [comment]
Ah I never hit this issue at runtime because everytime I called the getTotalFiat function I passed it an exchange rate. The LOC that was an issue is only run if getTotalFiat is not passed an exchangeRate.
